ChatBot/ChatBot/NetworkManager/NetworkManager.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  NetworkManager.swift
+//  ChatBot
+//
+//  Created by JaeHyeok Sim on 1/5/24.
+//
+
+import Foundation
+ 
+final class NetworkManager {
+    static let shared = NetworkManager()
+    
+    private init () {}
+
+    func makeRequest(builder: NetworkBuilderProtocol) -> URLRequest? {
+        
+        let baseUrl = BaseURL.openAiUrl
+        guard let url = URL(string: baseUrl + builder.path) else { return nil }
+      
+        var request = URLRequest(url: url)
+        
+        if builder.method.rawValue == "POST" {
```

Comment:
```suggestion
        if builder.method == .POST {
```

=> rawValue 말고  enum case로 비교해도 될 것 같아요~!
rawValue 로 직접 비교하면 문자열로 비교하는 것이니 실수가 발생할 수 있다고 생각합니다 ㅎㅎ

Reply:
말씀하신 대로 문자열로 비교를 하면 휴먼에러가 발생할 수 있다고 생각하여 enum case로 비교하는 것으로 수정하였습니다.